controllers/hibp.js

```diff
@@ -13,6 +13,20 @@ const mozlog = require("../log");
 const log = mozlog("controllers.hibp");
 
 
+// Get address and language from either subscribers or
+// email_addresses fields
+function getAddressAndLanguageForEmail(recipient) {
```

Comment:
There's probably a more stream-lined way to do this?

Reply:
Though maybe not, because we still want/need to add a "Send all breach alerts to my primary inbox" feature, and that logic is best added here.